system_metrics_collector/src/topic_statistics_collector/imu_talker.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <utility>
+
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/imu.hpp"
```

Comment:
Consider adding a simple dummy message type to this package, to avoid pulling a dependency to `sensor_msgs` just for this reason.

Reply:
Implemented in https://github.com/ros-tooling/system_metrics_collector/pull/121.